### DISCUSSION TITLE
Ability to Filter Resources Events

### DISCUSSION THREAD

**Discussion by muhammadsaeedparacha** [wdelfuego/nova-calendar#29] [Ideas & Feature Requests]
**Usage:**
When I view a calendar that shows me too many events
I can narrow down the list of events shown to me
To specific Nova Resources

*Possible Implementation*
Front-end has a multi-select/checkboxes for each of the Nova Resources
When I uncheck some checkboxes of resources
The backend filters out those resources from being displayed to me
And Saves the selection in local storage perhaps?

**Comment by redon92** [comment]
I am interested in this too. It would be great to have a way to filter the novaResources and nonNovaResources events.

**Comment by wdelfuego** [comment]
Thanks a lot for your input, @muhammadsaeedparacha and @redon92 !

I completely agree that this would be a worthwhile addition. I'm currently low on time for this but have thought about the implementation and I don't think it's too complicated so I'll be getting to it in an upcoming release.

Instead of filtering based on Nova resource class, I think I'd let the developer implement their own _event_ filters so that they can be as simple or complex as you want. That allows non-nova events to be filtered, too.

I think I'd show the available filters above the calendar, either as a single or a multi-select, in a way that doesn't limit the amount of filters that can be presented.

Let me know if you have any ideas, comments or suggestions with regard to this feature!

**Reply by scramatte** [reply]
Hi,

I would be great to have such feature!

**Comment by wdelfuego** [comment]
Event filtering has been released as part of the 2.0 release today :).

Please, create issues for any doubts you may have or issues you may run into 🙏.
